### DISCUSSION TITLE
wlan1 as client, wlan0 as hotspot; no Internet when hotspot enabled

### DISCUSSION THREAD

**Discussion by jonnyrcarey** [RaspAP/raspap-webgui#818] [Q&A] [answered]
I don't want to open an issue because I'm sure there's a way I can address this and/or I'm missing a step. I'm trying to use the Pi as either a repeater or a bridge. (No eth0 connection.)  I can get RaspAp up and running, but I can't get internet. I've searched the FAQ, closed issues, etc.

--New installation, headless.  Installed all via ssh and wifi using Quick Installer.  Not AP-STA mode.
--Hotspot works: I can connect to webGUI showing wlan0 up
--Dashboard also says that wifi client works and is active on wlan1

But internet doesn't work, neither on the Pi nor on connected devices.  Dashboard shows an initial spike of data transferred at installation hour, and then nothing.

However, if I turn off the hotspot, then the Internet works on the Pi itself!

Is this possibly related to iptables?  My upstream router is at 192.168.50.1 and its DHCP assigns in 192.168.50.2 to 192.168.50.254 range.  So, on the upstream network, wlan1 is at 192.168.50.168.  Is this creating a conflict with iptables or other defaults in RaspAp?  I tried editing iptables and other locations where I found 192.168.50.X in RaspAp files, but that didn't work.  And really, I'm out of my depth.

Also, I'm unsure if this is helpful, but if I try reassigning the hotspot to wlan1 and restart, then I can no longer connect to raspap-webgui, even though it's broadcasting.

Any suggestions/help greatly appreciated!

------
Rasp Pi 3B+ 
RaspOS Lite (also tested with Pixel desktop: same issue)
Edimax 7811Un wireless adapter (and duplicated on 2 other plug-and-play adapters)

**Comment by billz** [comment]
This thread on /r/RaspAP is relevant to your setup and might be helpful
https://www.reddit.com/r/RaspAP/comments/jnhky0/noob_question/

Re: accessing the UI, check this FAQ
https://docs.raspap.com/faq/#access

**Reply by jonnyrcarey** [reply, marked as answer]
Interesting thread.  Thank you.  I tried official power supply, but still no-go.  In the end, I changed to Rpi 4.  Still didn't work.  But I did get it to work by setting it via Ethernet cable, and THEN setting up WiFi client on wlan1.  Not ideal to have to do that extra step, but the point is that it now works, which was what I needed.

I appreciate your quick reply, your suggestion of issues, and this service you've created/manage.  I've set up three now--one at my home, and two each at my parents' and sisters' homes...all to provide wireless to the "dead zones" in the respective house.  We don't need some high-end mesh system--we just need enough WiFi for basic tasks in these dead zones, and RaspAp serves that purpose so well.

**Reply by billz** [reply]
Glad to hear it and thanks for sharing your solution.

**Comment by sn3ik** [comment]
Hi jonnyrcarey,

I also had the same problem.
To share the connection between the two networks you have to disable the static routers in /etc/dhcpcd.conf

interface wlan0 (wlan? <- your access point)
static ip_address = 10.3.141.1 / 24
#static routers = 10.3.141.1

**Reply by softdream1981** [reply]
It's work!!
